result/isislsp.yaml

```diff
@@ -0,0 +1,341 @@
+
+    IsisLsp.neighbor:
+      description: >-
+        This container describes IS neighbors.
+      type: object
+      properties:
+        system_id:
+          description: >-
+            The System ID for this emulated ISIS router, e.g. "640100010000".
+          type: string
+          format: hex
+
+    IsisLsp.Ipv4InternalReachability:
+      description: |-
+        This container defines list of IPv4 internal reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V4Prefix'
+
+    IsisLsp.Ipv4ExternalReachability:
+      description: |-
+        This container defines list of IPv4 external reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V4Prefix'
+
+    IsisLsp.V4Prefix:
+      description: >-
+        This group defines attributes of an IPv4 standard prefix.
+      type: object
+      properties:
+        ipv4_address:
+          description: >-
+            An IPv4 address.
+          type: string
+        redistribution_type:
+          description: |-
+            Up (0)-used when a prefix is initially advertised within the ISIS L3 hierarchy, 
+              and for all other prefixes in L1 and L2 LSPs. (default) 
+            Down (1)-used when an L1/L2 router advertises L2 prefixes in L1 LSPs. 
+            The prefixes are being advertised from a higher level (L2) down to a lower level (L1). 
+          type: string
+          enum: [up, down]
+        default_metric:
+          description: |-
+            ISIS default metric value.
+          type: integer
+          minimum: 0
+          maximum: 63
+        origin_type:
+          description: >-
+            The origin of the advertised route-internal or external to the ISIS area.
+            Options include the following: 
+              Internal-for intra-area routes, through Level 1 LSPs. 
+              External-for inter-area routes redistributed within L1, through Level 1 LSPs.
+          type: string
+          enum: [internal, external]
+  
+    IsisLsp.ExtendedIpv4Reachability:
+      description: |-
+        This container defines list of IPv4 extended reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            IPv4 prefix contained within extended reachability TLVs.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.ExtendedV4Prefix'
+
+    IsisLsp.ExtendedV4Prefix:
+      description: >-
+        This group defines attributes of an IPv4 standard prefix.
+      type: object
+      properties:
+        ipv4_address:
```

Comment:
Should this property have 'format : ipv4' in it?

Reply:
agreed.